src/Login/Login.jsx

```diff
@@ -66,40 +66,100 @@ class Login extends React.Component {
       : 'gene';
     const customImageStyle = { backgroundImage: `url(/src/img/icons/${customImage}.svg)` };
 
-    const loginOptions = {}; // one for each login provider
-    const filterOptions = {};
-    this.props.providers.forEach((provider, i) => {
+    let loginComponent = (
+      <React.Fragment key='login-component'>
+        <div className='login-page__entries'>
+          <div className='login-page__entry-login'>
+            <Button
+              onClick={() => {}}
+              label='Loading login options'
```

Comment:
i don't know if it was part of the design, but i think no text (only spinner) may look better

Reply:
i agree, i removed the text and make the button have the same width as enabled login buttons